docs/content/deployment/deploying_scheduler.md

```diff
@@ -36,42 +36,88 @@ steps to initialize configuration and start the Airflow scheduler.
 
 ### CeleryExecutor
 
-1. Prepare broker and backend. Celery needs a broker and a result backend to distribute and store tasks. For more
-   information about setting up the broker and result backend, refer to the exhaustive
-   [Celery documentation on the topic](https://docs.celeryproject.org/en/latest/getting-started/).
+#### Standalone Deployment
 
-2. Set environment variables
+1. Set up Celery broker and result backend. CeleryExecutor take advantage of the [Celery](https://docs.celeryproject.org/en/stable/index.html) to 
+   scale out the number of workers. For this to work, you need to setup a broker and a result backend (RabbitMQ, Redis, ...). 
```

Comment:
It's recommended to tell the users how to setup a broker and a result backend, which could give a link to users.